content/blog/2020/06/2020-06-03-git-performance-improvement-phase1.adoc

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Git Plugin Performance Improvement: Phase-1"
+tags:
+- plugins
+- platform-sig
+- developer
+- performance-improvement
+- gsoc
+- gsoc2020
+author: rishabhbudhouliya
+opengraph:
+  image: /images/post-images/2020/07-git-plugin-performance-improvement/gsoc-git-opengraph.png
+---
+
+Git Plugin Performance Improvement is a Google Summer of Code 2020 project.
+It aims to improve the performance of the git plugin which provides fundamental git functionalities.
+
+Internally, the plugin provides these functionalities using two implementations: cli git and jgit(pure java implementation).
+
+
+image:/images/post-images/2020/07-git-plugin-performance-improvement/git-intro.png[git-intro]
+*cli git is the default implementation for the plugin, a user can switch to jgit if needed*
+
+The project is divided into two parallel stages:
+
+* *Stage 1*: Create benchmarks which evaluate the execution time of a git operation provided by cli git and jgit using JMH, which is a micro benchmarking test harness.
```

Comment:
I would remove `which is` from `Stage 1*: Create benchmarks which evaluate the execution time of a git operation provided by cli git and jgit using JMH, which is a micro benchmarking test harness.`